README.md

```diff
@@ -52,4 +52,4 @@ Below are some projects created by Kolide community members. Please submit a pul
 
 Looking for the quickest way to try out osquery on your fleet? Not sure which queries to run? Don't want to manage your own data pipeline?
 
-Try our [osquery SaaS platform](https://kolide.com/?utm_source=oss&utm_medium=readme&utm_campaign=fleet) providing insights, alerting, fleet management and user-driven security tools. We also support advanced aggregation of osquery results for power users. Get started immediately, and your first 10 hosts are free.
+Try our [osquery SaaS platform](https://kolide.com/?utm_source=oss&utm_medium=readme&utm_campaign=fleet) providing insights, alerting, fleet management and user-driven security tools. We also support advanced aggregation of osquery results for power users. Get started with your 30-day free trial [today](https://kolide.com/signup).
```

Comment:
Please add the GA campaign tags to the link (`utm_source=oss&utm_medium=readme&utm_campaign=fleet`)